src/Microsoft.DocAsCode.Dotnet/SymbolFilter.cs

```diff
@@ -94,6 +94,20 @@ private bool IsSymbolAccessible(ISymbol symbol)
             if (symbol.IsImplicitlyDeclared && symbol.Kind is not SymbolKind.Namespace)
                 return false;
 
+            if (_config.IncludeExplicitInterfaceImplementations)
+            {
+                bool isEii = symbol.Kind switch
```

Comment:
consider extract a `bool SymbolHelper.TryGetExplicitInterfaceImplementations(ISymbol symbol, out IEnumerable<ISymbol> eiis)` method and reuse it in line 113